### PR TITLE
add realms actions

### PR DESCRIPTION
**Pull request by 0xlarry**

### CODE REVIEW

File: types/eventConfig/requirements.ts [hunk 1]
```diff
@@ -183,3 +183,13 @@ export const SignUpActionSchema = z.object({
...
  */
 export type SignUpAction = z.infer<typeof SignUpActionSchema>;
+
+/**
+ * REALMS VOTE ACTION
+ */
+export const RealmsVoteActionSchema = z.object({
+  daoPubKey: z.string(),
+  proposalPubKey: z.string(),
+  curstomProgramId: z.string().optional(),
```
Comment (nardevvv) [inline types/eventConfig/requirements.ts:193]: Just spelling of `curstomProgramId` here

Also, should this field be added in client?

File: types/events.ts [hunk 2]
```diff
@@ -47,6 +47,11 @@ export const OnChainEventRequestSchema = z.discriminatedUnion("eventType", [
     bidder: z.string(),
     amount: z.number(),
   }),
+  z.object({
```
Comment (nardevvv) [inline types/events.ts:50]: Should include all fields from above

File: types/eventConfig/requirements.ts (hunk 1 above)
Comment (0xlarry) [reply types/eventConfig/requirements.ts:193]: cusrtom lol ... yes eventually but I dont wanna add it until we support that... does that make senes?

File: types/events.ts (hunk 2 above)
Comment (0xlarry) [reply types/events.ts:50]: im not sure what you mean by this?

Comment (nardevvv) [reply types/events.ts:50]: I figured `customProgramId` should also be in the zod schema in events.ts here

**Pull request merged**